cmd/bom/cmd/generate.go

```diff
@@ -364,13 +385,34 @@ func generateBOM(opts *generateOptions) error {
 	if err != nil {
 		return fmt.Errorf("serializing document: %w", err)
 	}
-	if opts.outputFile == "" {
+
+	switch opts.outputFile {
+	case "":
 		fmt.Println(markup)
-	} else {
+	case "github":
+		matches := githubReleaseRe.FindStringSubmatch(opts.ghRelease)
+		if matches == nil {
+			return fmt.Errorf("wrong github org/repo@tag input for %s", opts.ghRelease)
+		}
+		org := matches[1]
+		repo := matches[2]
+		tag := matches[3]
+		tempFileName := repo + ".sbom"
```

Comment:
Let follow here the [openssf SBOM naming convention](https://github.com/ossf/sbom-everywhere/blob/main/reference/sbom_naming.md):
```suggestion
		jsonExt := ""
		if opts.format == "json" {
		      jsonExt = spdx.FormatJSON
		}
		tempFileName := fmt.Sprintf("%s-%s.spdx%s", repo, tag, jsonExt)
```